src/care/forall.h

```diff
@@ -75,9 +76,11 @@ namespace care {
       const int length = end - start;
 
       if (length != 0) {
-#ifndef CARE_DISABLE_RAJAPLUGIN
-         RAJAPlugin::pre_forall_hook(ExecutionPolicyToSpace<ExecutionPolicy>::value, fileName, lineNumber);
-#endif
+         ProfilePlugin::setFileName(fileName);
```

Comment:
I would make another class with shared plugin data that both the debug and profile plugin need, such as the file name and line number.